packages/application/src/test/application.spec.ts

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as nbformat from '@jupyterlab/nbformat';
+import { NotebookModel } from '@jupyterlab/notebook';
+import { JupyterServer, NBTestUtils } from '@jupyterlab/testutils';
+
+import { NotebookParser } from '../parsing';
+import { RequestHandler } from '../requests';
+import { FrontendServices } from '../services';
+
+const server = new JupyterServer();
+const defaultContent: any = {
+  cells: [
+    {
+      cell_type: 'code',
+      execution_count: null,
+      metadata: {},
+      outputs: [],
+      source: ['import os\n', "print(os.environ['HOME'])"]
```

Comment:
Since this is going to be used for testing the env variable detection logic, please add an additional line with this syntax - with double, instead of single quotes for difference:
```python
print(os.getenv("HOME2"))
```
Reading up on the [os methods](https://docs.python.org/3/library/os.html#process-parameters), we should probably add the "bytes" versions of these as well...
```python
print(os.environb['HOME_B'])
print(os.getenvb("HOME_B2"))
```
I'm suspect that test will fail. :smile: